admin/views/tabs/metas/post-types.php

```diff
@@ -17,18 +17,15 @@
 
 if ( is_array( $post_types ) && $post_types !== array() ) {
 	foreach ( $post_types as $pt ) {
-		$warn = false;
-		if ( $options['redirectattachment'] === true && $pt->name == 'attachment' ) {
-			echo '<div class="wpseo-warning">';
-			$warn = true;
-		}
 		$name = $pt->name;
 		echo "<div id='" . esc_attr( $name ) . "-titles-metas'>";
 		echo '<h2 id="' . esc_attr( $name ) . '">' . esc_html( ucfirst( $pt->labels->name ) ) . '</h2>';
-		if ( $warn === true ) {
-			echo '<h3 class="error-message">' . esc_html__( 'Take note:', 'wordpress-seo' ) . '</h3>';
-			echo '<p class="error-message">' . __( 'As you are redirecting attachment URLs to parent post URLs, these settings will currently only have an effect on <strong>unattached</strong> media items!', 'wordpress-seo' ) . '</p>';
-			echo '<p class="error-message">' . sprintf( __( 'So remember: If you change the %sattachment redirection setting%s in the future, the below settings will take effect for *all* media items.', 'wordpress-seo' ), '<a href="' . esc_url( admin_url( 'admin.php?page=wpseo_advanced&tab=permalinks' ) ) . '">', '</a>' ) . '</p>';
+		if ( $options['redirectattachment'] === true && $pt->name == 'attachment' ) {
```

Comment:
You could use `$name` here instead of `pt->name`.
Also, personally I'm not a fan of abbreviations. I had to check what `$pt` stands for. I'd prefer $post_type, but I'll leave it up to you to decide what's best.